fiftyone/core/models.py

```
@@ -1994,15 +2000,12 @@ def _patch_collate_fn(batch):
     return batch[0]  # return patches directly
 
 
-def _parse_batch_size(batch_size, model, use_data_loader):
+def _parse_batch_size(*, batch_size, model):
```

Ruff (0.14.10)

Missing return type annotation for private function `_parse_batch_size`

(ANN202)